extensions/amp-analytics/0.1/events.js

```diff
@@ -389,6 +425,42 @@ export class VisibilityTracker extends EventTracker {
   }
 
   /**
+   * @param {string|undefined} waitForSpec
+   * @param {string|undefined} selector
+   * @param {?Element} element
+   * @return {?Promise}
+   * @visibleForTesting
+   */
+  getReadyPromise(waitForSpec, selector, element) {
+    if (!selector) {
+      // Default case #1: wait for nothing if selector is not specified.
+      return null;
```

Comment:
Changed.
However I would still argue it is a bit unclear. Without `waitFor` `selector=undefined` and `selector=root` create different behaviors. With `waitFor` defined  `selector=undefined` `selector=root` are the same.  😕